main_app/views.py

```diff
@@ -0,0 +1,37 @@
+from django.shortcuts import render
+from django.views.generic import TemplateView
+
+from .igdb_api import IGDB
+
+
+class MainPageView(TemplateView):
+    template = 'main_page.html'
+
+    def __init__(self, **kwargs):
+        super(MainPageView, self).__init__(**kwargs)
+        self.client = IGDB(6)
+        self.__current_page = '1'
+        self.__search = ''
+        self.__platforms = ''
+        self.__genres = ''
+        self.__user_ratings = ''
+
+    def get_context_data(self, **kwargs):
+        super(MainPageView, self).get_context_data()
```

Comment:
Take a look at example from documentation, 
```
    def get_context_data(self, **kwargs):
        context = super().get_context_data(**kwargs)
        context['now'] = timezone.now()
        return context
```
you should get initial context dictionary and update it with your own values.
You can print value of `context` variable, There will be some useful variables set by Django. So you need to __extend__ this dict